mkdocs.yml

```diff
@@ -87,5 +87,7 @@ nav:
         - Application Registry: 'development/application-registry.md'
         - User Preferences: 'development/user-preferences.md'
         - Release Checklist: 'development/release-checklist.md'
+    - User Guides:
```

Comment:
Please move this section up so that it falls between `External Authentication` and `Core Functionality` above.